apps/client/src/features/search/components/search-result-item.tsx

```
@@ -0,0 +1,111 @@
+
+          <div style={{ flex: 1 }}>
+            <Text>{attachmentResult.fileName}</Text>
+            <Text size="xs" opacity={0.6}>
+              {attachmentResult.space.name} • {attachmentResult.page.title}
+            </Text>
+
+            {attachmentResult?.highlight && (
+              <Text
+                opacity={0.6}
+                size="xs"
+                dangerouslySetInnerHTML={{ __html: attachmentResult.highlight }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)